### PR TITLE
Vote transaction should update the votedDelegatesPublicKeys property for account - Closes #1128

### LINKED ISSUE #1128: Vote transaction should update the votedDelegatesPublicKeys property for account
### Expected behavior
The votes casted by an account are stored in the `votedDelegatePublicKeys` property.  Therefore, on applying a vote transaction, the vote transaction should update the `votedDelegatePublicKeys` property.
### Actual behavior
The Vote transaction updates the `votes` property for an account.
### Which version(s) does this affect? (Environment, OS, etc...)
v2.*

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

Account Votes where being stored in the `votes` property

### How did I fix it?

Changed the `votes` property to `votedDelegatesPublicKeys`

### How to test it?

Build should be green

### Review checklist

* The PR resolves #1128
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: packages/lisk-transactions/src/3_vote_transaction.ts [hunk 1]
```diff
@@ -300,20 +300,23 @@ export class VoteTransaction extends BaseTransaction {
 			.filter(vote => vote.charAt(0) === PREFIX_UNVOTE)
 			.map(vote => vote.substring(1));
 		const originalVotes = sender.votes || [];
```
Comment (SargeKhan) [inline packages/lisk-transactions/src/3_vote_transaction.ts]: Here it should be: `sender.votedDelegatesPublicKeys`

Comment (SargeKhan) [reply packages/lisk-transactions/src/3_vote_transaction.ts]: And i think this [line](https://github.com/LiskHQ/lisk-elements/pull/1129/files#diff-f9ecf85bc7239f90e9bef6de8faeb1deL280) is redundant.

Comment (pablitovicente) [reply packages/lisk-transactions/src/3_vote_transaction.ts]: Actually I might need to check all the usages of `sender.votes` and change to `votedDelegatesPublicKeys`

Comment (pablitovicente) [reply packages/lisk-transactions/src/3_vote_transaction.ts]: Had to also update `undoAsset` good catch!

**Review by shuse2 — APPROVED** [review]

**Review by mitsuaki-u — APPROVED** [review]

**Review by SargeKhan — APPROVED** [review]

**Pull request merged**